style.css

```diff
@@ -0,0 +1,37 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;1,100;1,200;1,300;1,400&display=swap");
```

Comment:
well done by adding font-family 👪
the role of font-family in a website 

Font-family refers to a collection of fonts that share similar design characteristics, such as size, weight, and style. When specifying a font-family in a web design, it is important to choose a font that is legible and appropriate for the content, style, and tone of the website.

The font-family property is used in CSS (Cascading Style Sheets) to specify the order of fonts that should be used to display text. If the first font is not available, the browser will move to the next one in the list until it finds a suitable font to display the text.